### PR TITLE
Don't use RTLD_NOLOAD alone in the dlopen calls

### PR DESCRIPTION
**Pull request by chinese-soup**
As per investigation in https://github.com/YaLTeR/BunnymodXT/issues/326, 

TLDR: `dlopen()` here fails and returns `NULL` on glibc 2.36+ (subsequently, `dlclose(thatNULLfromthedlopencall)` then segfaults), even though it should be more like `invalid mode for dlopen(): Invalid argument` (this should be true even for older glibc, so no idea why it worked until now, like I mentioned in the issue, maybe the libraries were already somehow loaded before, so it didn't matter with older versions?) but whatever. 

Like we mentioned in the Discord earlier, OR'ing it with RTLD_LAZY (or RTLD_NOW) fixes this and BXT injects fine and everything works flawlessly, so let's just do this?

### CODE REVIEW

**Comment by YaLTeR** [comment]
Yeah, sounds good, thanks for investigating.
